numpy/_core/tests/examples/cython/meson.build

```diff
@@ -14,6 +14,15 @@ npy_include_path = run_command(py, [
     'import os; os.chdir(".."); import numpy; print(os.path.abspath(numpy.get_include()))'
     ], check: true).stdout().strip()
 
+npy_path = run_command(py, [
+    '-c',
+    'import os; os.chdir(".."); import numpy; print(os.path.dirname(numpy.__file__).removesuffix("numpy"))'
+    ], check: true).stdout().strip()
+
+# TODO: This is a hack due to gh-25135, where cython may not find the right
+#       __init__.pyd file.
+add_project_arguments('-I', npy_path, language : 'cython')
```

Comment:
This was the best way I found, I tested it locally with an in-place build at least...

Reply:
What if you unconditionally add the current working directory? For any other use case, I think the current Cython logic does the right thing.

Reply:
OTOH, I see the lines above also use the `cd ..` trick in order to get the C include path, so this is reusing that hack.

Reply:
I don't have a feeling on what is best here.  Wasn't this run in a temporary folder though?  Escaping that might be annoying.